daml-lf/spec/daml-lf-1.rst

```diff
@@ -719,12 +719,12 @@ Then we can define our kinds, types, and expressions::
     u ::= 'pure' @τ e                               -- UpdatePure
        |  'bind' x₁ : τ₁ ← e₁ 'in' e₂               -- UpdateBlock
   x     |  'create' @Mod:T e                         -- UpdateCreate
-       |  'create_interface' @Mod:I e               -- UpdateCreateInterface [Daml-LF ≥ 1.dev]
+       |  'create_interface' @Mod:I e               -- UpdateCreateInterface [Daml-LF ≥ 1.15]
        |  'fetch' @Mod:T e                          -- UpdateFetch
-       |  'fetch_interface' @Mod:I e                -- UpdateFetchInterface [Daml-LF ≥ 1.dev]
+       |  'fetch_interface' @Mod:I e                -- UpdateFetchInterface [Daml-LF ≥ 1.15]
   x     |  'exercise' @Mod:T Ch e₁ e₂                -- UpdateExercise
   x     |  'exercise_by_key' @Mod:T Ch e₁ e₂         -- UpdateExerciseByKey [Daml-LF ≥ 1.11]
-  x     |  'exercise_interface' @Mod:I Ch e₁ e₂ e₃   -- UpdateExerciseInterface [Daml-LF ≥ 1.dev]
+  x     |  'exercise_interface' @Mod:I Ch e₁ e₂ e₃   -- UpdateExerciseInterface [Daml-LF ≥ 1.15]
```

Comment:
Maybe you could drop those `x` that I probably merged by accident.

Reply:
ok, will do... I thought they might have meant something!